calico-enterprise_versioned_docs/version-3.15/networking/egress/egress-gateway-aws.mdx

```diff
@@ -770,6 +770,10 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: status.podIP
+         # Optional: Used to specify IP tables backend (legacy or nft). If not specified, egress gateway will auto-detect.
+         # If both nft and legacy are supported, nft will be used.
+        - name: IPTABLES_BACKEND
+          value: "nft"
```

Comment:
We shouldn't default to `nft` in the docs.  Users will copy-paste without checking every flag is correct.  Better to have optional parameters commented out in some fashion, I think.

Reply:
Worth saying "for example on RHEL 9" or something for when you might want to use nft.